libs/watt/package/dropdown/watt-dropdown.component.ts

```diff
@@ -208,54 +179,75 @@ export class WattDropdownComponent implements ControlValueAccessor, OnInit {
     effect(() => {
       const options = this.options();
       if (Array.isArray(options)) {
-        let optionsCopy = [...options];
+        const optionsCopy = [...options];
 
-        if (this.sortDirection()) {
-          optionsCopy = this.sortOptions(optionsCopy);
-        }
+        const hasGroups = options.some(
+          (option) =>
+            'options' in option && Array.isArray((option as WattDropdownOptionGroup).options)
+        );
+
+        this.hasGroups.set(hasGroups);
```

Comment:
suggestion: `hasGroups` could be a `computed` signal instead

Reply:
Good idea, fixed